frame/nomination-pools/runtime-api/src/lib.rs

```diff
@@ -24,10 +24,19 @@ use codec::Codec;
 
 sp_api::decl_runtime_apis! {
 	/// Runtime api for accessing information about nomination pools.
+	#[api_version(1)]
```

Comment:
no need for this. This is only needed if on the rust client side we need to reuse the code. Any particular reason you have it here?

Reply:
Not really, I just thought it was a good practice to version the runtime APIs. Removing it.